docs/technical_reference/flowsheets/seawater_RO_desalination.rst

```diff
@@ -0,0 +1,200 @@
+Seawater RO Desalination
+========================
+
+Introduction
+------------
+
+This flowsheet represents a full-scale seawater reverse osmosis treatment facility.
+The flowsheet includes pretreatment, desalination, post-treatment, and waste handling unit processes available in WaterTAP.
+Unit models used on this flowsheet span the spectrum from simple to detailed.
+This flowsheet includes examples of several different types of modeling features available in WaterTAP, summarized in Table 1:
+
+.. csv-table::
+   :header: "Modeling Component", "Documentation"
+   
+   "Zero order unit and costing models", ":doc:`/technical_reference/unit_models/zero_order_unit_models/index`"
+   "Zero order costing package", ":doc:`/technical_reference/costing/zero_order_costing`"
+   "Zero order property package", ""
+   "Seawater property package", ":doc:`/technical_reference/property_models/seawater`"
+   "Reverse osmosis model", ":doc:`/technical_reference/unit_models/reverse_osmosis_0D`"
+   "Pump model", ""
+   "Energy recovery device model", ""
+   "Pressure exchanger model", ":doc:`/technical_reference/unit_models/pressure_exchanger`"
+   "WaterTAP costing package", ":doc:`/technical_reference/costing/watertap_costing`"
+   "Unit model costing packages", ":doc:`/technical_reference/costing/detailed_unit_model_costing`"
+   "WaterTAP database", ""
+   "IDAES Translator blocks", :doc:`idaes:reference_guides/model_libraries/generic/unit_models/translator`
+   "IDAES Product blocks", :doc:`idaes:reference_guides/model_libraries/generic/unit_models/product`
+   "IDAES Separator blocks", :doc:`idaes:reference_guides/model_libraries/generic/unit_models/separator`
+   "IDAES Mixer blocks", :doc:`idaes:reference_guides/model_libraries/generic/unit_models/mixer`
+
+The influent conditions are defined from the case study this flowsheet is modeled after. The relevant information is in Table 2:
+
+.. csv-table::
+   :header: "Description", "Value", "Units"
+
+   "Volumetric flow rate", "7.05", ":math:`\text{MGD}`"
+   "TDS concentration", "35", ":math:`\text{g/L}`"
+   "TSS concentration", "0.03", ":math:`\text{g/L}`"
+   "Temperature", "298", ":math:`\text{K}`"
+   "Pressure", "100000", ":math:`\text{Pa}`"
+
+Some unit models have case-specific operating conditions, presented in Table 3 and includes the different build options for ``erd_type``:
+
+.. csv-table::
+   :header: "Description", "Value", "Units", "Flowsheet Model Name"
+
+   **Pre-Treatment**
+   "Ferric chloride dose", "20", ":math:`\text{mg/L}`", "``m.fs.pretreatment.ferric_chloride_addition``"
+   "Storage tank 1 storage time", "2", ":math:`\text{hr}`", "``m.fs.pretreatment.storage_tank_1``"
+   
+   **Desalination**
+   "RO water permeability coeff.", "4.2E-12", ":math:`\text{m/Pa/s}`", "``m.fs.desalination.RO``"
+   "RO salt permeability coeff.", "3.5e-8", ":math:`\text{m/s}`", "``m.fs.desalination.RO``"
```

Comment:
```suggestion
   "RO water permeability coefficient", "4.2E-12", ":math:`\text{m/Pa/s}`", "``m.fs.desalination.RO``"
   "RO salt permeability coefficient.", "3.5e-8", ":math:`\text{m/s}`", "``m.fs.desalination.RO``"
```
I would just spell these out, especially because the text box is already large enough to fit the whole word